query_test.go

```diff
@@ -576,6 +576,11 @@ func TestQueryEffectiveBufferSizeHeterogeneous(t *testing.T) {
 	colsWrite := []int64{1, 1, 2}
 	a1DataWrite := []byte("abbccc")
 	a1OffWrite := []uint64{0, 1, 3}
+	a2DataWrite := []byte("abbcccdddd")
```

Comment:
`a2` and `a3` are missing from the schema.

Reply:
Added